version/generate_version

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# generates the version of file based on current git version
+
+cd $GOPATH/src/github.com/contiv/netplugin/version
+
+USE_RELEASE=$1
+
+if [ -z "${USE_RELEASE}" ]
+then
+  file_name=version_default.go
+  UseVersionStr="// +build !use_version" 
+  useDefaultVersionString="False"
+else
+  file_name=version_release.go
+  UseVersionStr="// +build use_version" 
+  useDefaultVersionString="True"
+fi
+
+GitCommitStr=`git show | grep commit | cut -f 2 -d ' ' | cut -b 1-10`
+VersionStr=`cat CURRENT_VERSION`
```

Comment:
We should add something like `-dirty` or `-unsupported` if there are any local changes on top of what's found in git.